src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -23,14 +22,6 @@ public class SwerveConfig {
 
     public static final double DEFAULT_MAX_DRIVE_SPEED_FPS = 2.5;
 
-    // low-level configuration for drive motors
-    // TODO replace PID parameters with ones tuned for our build
-
-    public static final NeutralMode DEFAULT_DRIVE_NEUTRAL_MODE = NeutralMode.Brake;
-    public static final double DEFAULT_DRIVE_RAMP_RATE = 0.3;
-    public static final PIDConstant DRIVE_PID = new PIDConstant(
-        0.1, 0.01, 5.0, 1023.0 / 20660.0, 300.0, -1.0, 1.0);
-
```

Comment:
Here I'm just removing some unused settings - we've moved this stuff into the factory method, so we don't need it here.